actions/plugins/trufflehog/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Plugins - Trufflehog secrets scanning
+description: Scans plugin zip files for secrets using Trufflehog.
+
+inputs:
+  folder:
+    description: |
+      Folder containing plugin zip files to scan.
+      It will be scanned recursively.
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install Trufflehog
+      shell: bash
+      run: |
+        curl -sSfL https://raw.githubusercontent.com/trufflesecurity/trufflehog/main/scripts/install.sh | sh
```

Comment:
I decided to use `main` here instead of tagged version. In my opinion, there's no need to pin it to specific version (unlike Mage, golangci-lint, etc) because trufflehog attempts to update to the latest version by default when launched anyways/